README.md

```diff
@@ -1,21 +1,74 @@
-# oni-demo
-Download a demo version of Open Network Insight, which can be run standalone on a windows laptop using Winpython https://sourceforge.net/projects/winpython/files/latest/download 
+# **Open Network Insight Demo**
 
+ONI-Demo can be downloaded and run standalone on a **Windows Laptop** 
 
-[Instructions for downloading and running the demo] 
+## **Demo SW Requirements**
 
-We use a portable version of python that has all the dependencies and stuff included, you need to download and run this first: 
-http://sourceforge.net/projects/winpython/files/latest/download 
+This demo requires Winpython, which can be downloaded here:
 
-Install it in the root directory as C:\winpython to avoid any awkwardness in file path references. 
+[Winpython Download][1]
+[1]:  https://sourceforge.net/projects/winpython/files/latest/download "Winpython Installer"  
 
-You want to unzip oni-demo-win.zip into the winpython/notebooks folder 
+Note: Install Winpython under root directory as **C:\winpython**    
 
-Run the shortcut "Jupyter Notebook.exe"�  in the WinPython folder.
 
-When you start the notebook server (above), you can access these two links:
+## **Downloading Demo**
 
-http://localhost:8888/files/oni-demo-win/index_sconnects.html
+In order to download Demo code - you can use Download ZIP option from this repository
 
-http://localhost:8888/files/oni-demo-win/storyboard_sconnect.html
+![Alt][2]
 
+[2]: docs/download-zip.bmp "Download ZIP File"
+
+Unzip the file *oni-demo-win.zip* into **C:\winpython\notebooks** folder
+
+## **Running Demo**
+
+In order to run ONI demo, start **Jupyter Notebook.exe** that is located under winpython folder
+
+A Web browser will be opened in the following path
+
+![Alt][3]
+
+[3]: docs/JupyterStartup.bmp "Jupyter Initial Startup"
+
+### *Opening Suspicious Connects Web Page*
+
+Copy and paste the following link into your browser
+
+    http://localhost:8888/files/oni-demo-win/index_sconnects.html
+
+Index_sconnects page looks similar to the image below
+
+![Alt][4]
+
+[4]: docs/Index_sconnects.bmp "Index Sconnects Web Page"
+
+In the Demo, which contains similar functionality to Open Network Insight User Interface, you can:
+
+  * Select rows in Suspicious Connects Frame
+    * Icons for Reputation Services & Geolocation examples
+  * Move & Select Network View objects
+    * Displaying Chord Diagrams
+  * Display Detail View information (by selecting a Suspicious Connect row)
+  * Use IP Address Filter
+  * Running Edge Investigation notebook
+
+### *Opening Storyboard Web Page*
+
+Copy and paste the following link into your browser
+
+    http://localhost:8888/files/oni-demo-win/storyboard_sconnect.html
+
+Storyboard page looks similar to the image below
```

Comment:
When opened with IE 11, it does not look like the image below.

It looks like the image below (on my system) when opened with Chrome.